yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html

```diff
@@ -19,14 +19,14 @@
       <div class= "border">
       {% if edit %}
         <div class="edit_button">
+        <form method="get" action="{% url 'edit_page' workflow.id version.id %}">
+            {% csrf_token %}
+            <button type="submit"> Edit Workflow</button>
+        </form>
+        <div>
```

Comment:
do we want to take away the "delete" class? or are we going to use that same style you used in your other merge request later on?

Reply:
yes I looked and the only thing associated with it is a font size